pkgs/os-specific/linux/dracut/default.nix

```diff
@@ -0,0 +1,100 @@
+{ stdenv
+, lib
+, fetchurl
+, makeWrapper
+, pkg-config
+, kmod
+, asciidoc
+, libxslt
+, docbook5
+, docbook_xsl
+, docbook_xsl_ns
+, coreutils
+, utillinux
+, gnused
+, gnugrep
+, squashfsTools
+, cpio
+, binutils
+, gzip
+, bzip2
+, lz4
+, lzop
+, zstd
+, xz
+}:
+
+stdenv.mkDerivation rec {
+  name = "dracut";
```

Comment:
why name and not pname?

Reply:
`pname` is just name right and then combined with `version` right?